libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -2487,6 +2549,43 @@ bool AP_AHRS_NavEKF::get_variances(float &velVar, float &posVar, float &hgtVar,
     return false;
 }
 
+bool AP_AHRS_NavEKF::yaw_alignment_complete(void) const
```

Comment:
this function seems to be completely unused? we don't usually add unused APIs

Reply:
Was used during initial development. Will remove.